webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -66,8 +67,11 @@ export default class CreateIssueModal extends PureComponent {
 
         this.props.create(issue).then((created) => {
             if (created.error) {
+                let errMessage = created.error.message;
+                errMessage = getErrorMessage(created.error.message).message;
```

Comment:
This should work with the change requested for `getErrorMessage`

```suggestion
                const errMessage = getErrorMessage(created.error.message);
```